utils/remote_control/navigator_emergency_control/navigator_emergency_control/mainpage.dox

```diff
@@ -0,0 +1,14 @@
+/**
+\mainpage
+\htmlinclude manifest.html
+
+\b navigator_emergency_control 
+
+<!-- 
+Provide an overview of your package.
```

Comment:
This documentation file is pretty barren, could you please either add a bit more to it or remove it? Your readme file was fairly comprehensive, so I think this can be removed.